### PR TITLE
People redux

### PR DESCRIPTION
**Pull request by jakubkedzierski92**
saga for descirption is ready now. The people details page is fixed with number 3. Now ill be working on routing

### CODE REVIEW

File: src/features/Description/Projects/getProjects.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const getProjects = async () => {
+
+    const response = await fetch("https://api.themoviedb.org/3/person/3/movie_credits?api_key=9515ffc857c67f1558538dad140abb29&language=en-US")
```
Comment (Kubabogd) [inline src/features/Description/Projects/getProjects.js:3]: Everything is good and you get data but you have set idPeople=3 and it always be Harrison Ford. I think you could use useParam(). It helped me in movieDetails

**Review by BogdanskiJ — APPROVED** [review]
The code works. You'll probably change something else, but overall the code works. Great job

File: src/features/Description/Projects/getProjects.js (hunk 1 above)
Comment (Werka481) [reply src/features/Description/Projects/getProjects.js:3]: I confirm information from Kuba. Details for only one character are displayed.

File: src/features/Description/index.js [hunk 2]
```diff
@@ -1,52 +1,48 @@
...
-      <Projects />
-      </Wrapper>
+      <Background>
+        <Wrapper>
+          <Tile>
+            <Photo src={url_img + details.profile_path}></Photo>
+            <Info>
+              <Name>{details.name}</Name>
+              <Information
+                birthday={details.birthday}
```
Comment (Werka481) [inline src/features/Description/index.js:36]: Date format is not compliant with the design.

File: src/features/Description/Projects/index.js [hunk 3]
```diff
@@ -35,19 +27,17 @@ export const Projects = () => {
...
               key={crewDetail.id}
               movieTitle={crewDetail.title}
               movieRating={crewDetail.vote_average}
               votesNumber={crewDetail.vote_count}
             />
           ))}
-          <Movie />
         </TileWrapper>
       </Wrapper>
     </>
```
Comment (Werka481) [inline src/features/Description/Projects/index.js:43]: Movie posters are not showing up.

**Review by Werka481 — APPROVED** [review]

**Pull request closed without merging**